server/app/repository.py

```diff
@@ -331,6 +339,24 @@ def count_stations(self) -> int:
         cursor.execute(q)
         return cursor.fetchone()["count"]
 
+    @use_cursor
+    def owned_stations(self, user_id: int) -> Stations:
+
+        q = ("SELECT s.name, s.station_id "
+             "FROM stations s, station_owners "
```

Comment:
Use the JOIN clause instead of FROM-WHERE.

Reply:
rewritten. Is it ok?